src/vs/workbench/contrib/notebook/browser/contrib/outline/notebookOutline.ts

```diff
@@ -549,6 +549,11 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 			default: false,
 			markdownDescription: localize('outline.showCodeCells', "When enabled notebook outline shows code cells.")
 		},
+		'notebook.outline.onlyShowMarkdownHeaders': {
```

Comment:
I wonder if this can be something like "showNonHeaderMarkdownCells" to align with "showCodeCells", and later "showCodeSymbols".